cmd/client-fs.go

```diff
@@ -193,6 +193,20 @@ func (f *fsClient) Watch(params watchParams) (*watchObject, *probe.Error) {
 	}, nil
 }
 
+func shouldOverrideSuffix(objectPath string) bool {
```

Comment:
Can you rename this to isCharDevice() or isStreamFile(), it would be easier understand why we are using this.